### PR TITLE
Added initial integration test template.

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: integrations
```
Comment (picklelo) [inline .github/workflows/integration.yml:1]: nit: shall we call it `integration` to match the file name?

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      uses: actions/cache@v2
+      with:
+        path: .venv
+        key: python-${{ matrix.python-version }}-pydeps-${{ hashFiles('**/poetry.lock') }}
+
+    # Begin the integration test
+    - run: poetry install --no-interaction --no-root
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+    - run: ls -R
+    - name: Poetry Install
```
Comment (picklelo) [inline .github/workflows/integration.yml:63]: I think this `poetry install` is repeated twice. We may only need the first one, followed by the `pip install` below.

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      id: cache-deps
+      uses: actions/cache@v2
+      with:
+        path: .venv
+        key: python-${{ matrix.python-version }}-pydeps-${{ hashFiles('**/poetry.lock') }}
+
+    # Begin the integration test
+    - run: poetry install --no-interaction --no-root
+      if: steps.cache-deps.outputs.cache-hit != 'true'
+    - run: ls -R
```
Comment (picklelo) [inline .github/workflows/integration.yml:62]: Do we need this?

File: .github/workflows/integration.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      run: poetry install --no-interaction
+    - name: Install Requirements
+      working-directory: ./pynecone-website/pcweb
+      run: poetry run pip install -r requirements.txt
+    - name: Init Website
+      working-directory: ./pynecone-website/pcweb
+      run: poetry run pc init
+    - name: Run Website
+      working-directory: ./pynecone-website/pcweb
+      run: timeout 1m poetry run pc run || exit 0
```
Comment (picklelo) [inline .github/workflows/integration.yml:74]: So will this return a success if the server starts or how does it work?

Comment (Alek99) [reply .github/workflows/integration.yml:74]: If it doesn't error after a minute running it shuts this step dow

File: .github/workflows/integration.yml (hunk 2 above)
Comment (Alek99) [reply .github/workflows/integration.yml:63]: ah yes ill get rid of it duplicated

File: .github/workflows/integration.yml (hunk 1 above)
Comment (Alek99) [reply .github/workflows/integration.yml]: Ill name it to integration-test

**Review by picklelo — APPROVED** [review]
Looks good to me

**Pull request merged**